.github/CODEOWNERS

```diff
@@ -108,8 +108,9 @@
 # PRLabel: %Search
 /sdk/search/ @xirzec @sarangan12
 
-/sdk/applicationinsights/applicationinsights-query/ @divyajay @alongafni
-/sdk/operationalinsights/loganalytics/ @divyajay @alongafni
+# PRLabel: %Monitor - ApplicationInsights
+# ServiceLabel: %Monitor - ApplicationInsights %Service Attention
+/sdk/applicationinsights/Microsoft.Azure.App*/  @omziv @anatse @raronen @ischrei @danhadari @azmonapplicationinsights
```

Comment:
@omziv would you be the primary contact for any build failures in applicationinsights? One (not yet implemented) path I'd like to take is to have the primary contact be the first name listed and this is who I would assign any nightly build errors to - just as an FYI

Reply:
Actually nevermind, I see the packages under applicationinsights would not apply here.

Reply:
Regarding the primary contact - The path you're suggesting is not a favorable one for us. We have a weekly on call shift, who will be best suited to take care of any incidents that should occur. This contact can be found in the On-Call page of the IcM site.